app/components/cases/new-case.hbs

```diff
@@ -1,49 +1,47 @@
 {{! TODO: octane-refactor }}
```

Comment:
Mag deze uiteindelijk dan ook weg of mis ik hier nog iets dat gerefactored moet worden?

Reply:
ziet er refactored uit, mag wel weg